### PR TITLE
Enhanced UI of Blog Page

### PR DESCRIPTION
**Pull request by Anjuli08**
## Related Issue #261 

- Info about the related issue :-
The issue was to enhance the UI of blog page which I created a while back
- [x] SSOC Participant  
- [x] Contributor

Closes: #261 

### Describe the changes you've made
I have changed the color theme of the **whole page** including **navbar** to match with that of main page.
I have enhanced the **Back to home** button and improved hover effect on it and added a home icon to it.
Then I changed the **Cards** styling to decent one and improved the hovering effect on them to a better one.
I modified the color and hover on the **Read more** button.
Lastly I modified the **Scroll up button** in the footer.
## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I have tested all the changes in my local file and it worked well.

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |
 Original Screenshot |
iot blog (2)_LI

 Updated Screenshot |
iot '

iot2'

iot3'

### CODE REVIEW

**Review by mdnazisharman2803 — APPROVED** [review]
@prathimacode-hub  good to merge

File: assets/blog.html [hunk 1]
```diff
@@ -19,157 +19,160 @@
...
+                ('.nav-bar').addClass('floatingNav');
+            } else {
+                ('.nav-bar').removeClass('floatingNav');
+            }
+        });
+    </script>
+
+</body>
+
+</html>
```
Comment (Harshil-Jani) [inline assets/blog.html:178]: Add new line at EOF

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]
please add new lines at End of files as commented.

**Review by Anjuli08 — COMMENTED** [review]
Added comment in blog.html file

**Review by Anjuli08 — COMMENTED** [review]
done!

**Comment by mdnazisharman2803** [comment]
@Anjuli08  there is merge conflict... You  should resolve it

**Review by mdnazisharman2803 — APPROVED** [review]

**Review by Harshil-Jani — APPROVED** [review]

**Pull request merged**